connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java

```diff
@@ -96,6 +99,26 @@ public static void validate(Map<String, String> props) {
             throw new ConfigException("Must configure one of " +
                 SinkTask.TOPICS_CONFIG + " or " + SinkTask.TOPICS_REGEX_CONFIG);
         }
+
+        if (hasDlqTopicConfig) {
+            String dlqTopic = props.get(DLQ_TOPIC_NAME_CONFIG).trim();
+            if (hasTopicsConfig) {
+                String[] topics = props.get(SinkTask.TOPICS_CONFIG).split(",");
+                Arrays.setAll(topics, i -> topics[i].trim());
+                if (Arrays.asList(topics).contains(dlqTopic)) {
```

Comment:
I think we can simplify this logic using a utility method provided by the `ConfigDef` class:
```suggestion
                List<String> topics = ConfigDef.parseType(SinkTask.TOPICS_CONFIG, props.get(SinkTask.TOPICS_CONFIG), ConfigDef.Type.LIST);
                if (topics.contains(dlqTopic)) {
```